packages/owl-bot/src/owl-bot.ts

```diff
@@ -147,13 +151,41 @@ export function OwlBot(
     }
   });
 
+  // Ensure up-to-date configuration is stored on merge
+  app.on('pull_request.merged', async context => {
+    const configStore = new FirestoreConfigsStore(db!);
+    const installationId = context.payload.installation?.id;
+    const org = context.payload.organization?.login;
+
+    logger.info("Updating repo's configs via `pull_request.merged`");
+
+    if (!installationId || !org) {
+      logger.warn(`Missing install id (${installationId}) or org (${org})`);
```

Comment:
Let's make this a logger.error().  It means Owl Bot is significantly malfunctioning and the person on Bilge Pump duty should take a look.